lib/removeUnusedElements.js

```diff
@@ -0,0 +1,235 @@
+'use strict';
+var Cesium = require('cesium');
+var ForEach = require('./ForEach');
+var hasExtension = require('./hasExtension.js');
+
+var defined = Cesium.defined;
+
+module.exports = removeUnusedElements;
+
+/**
+ * Removes unused elements from gltf.
+ * This function currenlty only works for accessors, buffers, and bufferViews.
+ *
+ * @param {Object} gltf A javascript object containing a glTF asset.
+ * @returns {Object} The glTF asset with the added pipeline extras.
+ *
+ * @private
+ */
+function removeUnusedElements(gltf) {
+    removeUnusedElementsByType(gltf, 'accessor');
+    removeUnusedElementsByType(gltf, 'bufferView');
+    removeUnusedElementsByType(gltf, 'buffer');
+    return gltf;
+}
+
+/**
+ * Contains functions for converting a type to a glTF element name.
+ * @constructor
+ */
+function TypeToGltfElementName() {}
+
+TypeToGltfElementName.accessor = function() {
+    return 'accessors';
+};
+
+TypeToGltfElementName.buffer = function() {
+    return 'buffers';
+};
+
+TypeToGltfElementName.bufferView = function() {
+    return 'bufferViews';
+};
```

Comment:
`TypeToGltfElementName` can be simplified to

```
var TypeToGltfElementName = {
    accessor : 'accessors',
    buffer : 'buffers',
    bufferView : 'bufferViews'
};
```

Reply:
Done.